### ISSUE TITLE
Don't require login for "common" things

### ISSUE THREAD

**Issue by kosarko** [ufal/lindat-kontext#102]
@stranak anything that really needs to be login only, or do you wan't everything available?

**Comment by Ansa211** [comment]
I think the reason UCNK has it this way is to force users to log in (and then report the number of users etc. to grant agencies).

**Comment by stranak** [comment]
> On 12 Sep 2017, at 13:10, Anna Vernerová <notifications@github.com> wrote:
> 
> I think the reason UCNK has it this way is to force users to log in (and then report the number of users etc. to grant agencies).
> 
> 
Yes, but we have our statistics. Also, UCNK at least lets you know, what is going on, we don't and it is not obvious at all.

--p

**Comment by kosarko** [comment]
2017-09-12 13:50 GMT+02:00 Pavel Stranak <notifications@github.com>:

>
> > On 12 Sep 2017, at 13:10, Anna Vernerová <notifications@github.com>
> wrote:
> >
> > I think the reason UCNK has it this way is to force users to log in (and
> then report the number of users etc. to grant agencies).
> >
> >
> Yes, but we have our statistics. Also, UCNK at least lets you know, what
> is going on, we don't and it is not obvious at all.
>

There's a popup saying something about limited functionality when you
execute query as anonymous user. Does UCNK do more?

**Comment by stranak** [comment]
> On 12 Sep 2017, at 17:31, Ondřej Košarko <notifications@github.com> wrote:
> 
> There's a popup saying something about limited functionality when you
> execute query as anonymous user. Does UCNK do more?

No, that is what UCNK does. We didn't do it in the old Kontext. If we do it now, it is better, but I would still prefer to enable full functionality by default.

--p
